lib/OpenQA/WebAPI/Plugin/ObsRsync.pm

```diff
@@ -515,4 +536,26 @@ sub _for_every_batch {
     return @ret;
 }
 
+# Based on https://www.suse.com/c/multi-factor-authentication-on-suses-build-service/
+sub _bs_ssh_sign ($key, $realm, $value) {
+
+    # This needs to be a bit portable for CI testing
+    my $tmp = Mojo::File::tempfile->spew($value);
+    my @lines = split "\n", qx/ssh-keygen -Y sign -f "$key" -q -n "$realm" < $tmp/;
```

Comment:
Likely we should add an explicit dependency for `openssh-common` in `dependencies.yaml`.

Maybe possible errors should also be handled (see https://perldoc.perl.org/perlop#Quote-Like-Operators for an example from the official documentation).

Reply:
See a9f3e19e4, added a small check for the key to exist before attempting to use it.

Reply:
I don't think it's a good idea to add openssh-common as a common requirement for everyone. I would treat it as optional. How about just using a "Recommends" and in here check for existance of ssh-keygen, fail with an explicit error message otherwise

Reply:
Addressed [c80a3a1](https://github.com/os-autoinst/openQA/pull/5360/commits/c80a3a19dc490d73f65e5207690b84fe312f6696).